From 9c6971570d78d36a200ef89598b5be36db3db25f Mon Sep 17 00:00:00 2001
From: GuoYong Zheng <zhenggy () chinatelecom ! cn>
Date: Fri, 17 Jun 2022 10:28:04 +0000
Subject: [PATCH 24/28] bfq: Remove useless code in bfq_lookup_next_entity

It is no need to judge entity is null or not here,
directly return entity is ok, so remove it.

Signed-off-by: GuoYong Zheng <zhenggy@chinatelecom.cn>
---
 block/bfq-wf2q.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/block/bfq-wf2q.c b/block/bfq-wf2q.c
index 3a3908c55..ea9ea0459 100644
--- a/block/bfq-wf2q.c
+++ b/block/bfq-wf2q.c
@@ -1452,9 +1452,6 @@ static struct bfq_entity *bfq_lookup_next_entity(struct bfq_sched_data *sd,
 			break;
 	}
 
-	if (!entity)
-		return NULL;
-
 	return entity;
 }
 
-- 
2.36.1.203.g1bcf4f6271

